django/contrib/postgres/functions.py

```diff
@@ -1,9 +1,18 @@
-from django.db.models import DateTimeField, Func, UUIDField
+import warnings
 
+from django.db.models import DateTimeField, Func
+from django.db.models.functions import UUID4
+from django.utils.deprecation import RemovedInDjango70Warning, django_file_prefixes
 
-class RandomUUID(Func):
-    template = "GEN_RANDOM_UUID()"
-    output_field = UUIDField()
+
+class RandomUUID(UUID4):
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
```

Comment:
Reminder for when you've finalized the deprecation strategy: Please add a `# RemovedInDjango70Warning` with instructions about how to alter the code.